src/Illuminate/Testing/TestView.php

```diff
@@ -106,6 +106,16 @@ public function assertViewMissing($key)
         return $this;
     }
 
+    /**
+     * Assert that the view's rendered content is empty.
+     */
+    public function assertViewEmpty()
+    {
+        PHPUnit::assertEmpty($this->rendered);
+
+        return true;
```

Comment:
Shouldn't this return $this not return true?

Reply:
So you can chain it i mean?

Reply:
Yeah - my bad. I actually fixed this in a second commit but clearly forgot to push it to the branch. I've made a PR to fix the return value.